pageserver/src/tenant/mgr.rs

```diff
@@ -2790,7 +2791,7 @@ pub(crate) async fn immediate_gc(
 
                 if let Some(rtc) = rtc {
                     // layer drops schedule actions on remote timeline client to actually do the
-                    // deletions; don't care just exit fast about the shutdown error
+                    // deletions; don't care about the shutdown error, just exit fast
```

Comment:
```suggestion
                    // deletions; don't care about the shutdown error, just exit quickly
```

alternatively: "just do a fast exit"

Reply:
I don't know why this was an important suggestion.